pkgs/tools/system/acpica-tools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, bison, flex }:
+
+stdenv.mkDerivation rec {
+  name = "acpica-tools-${version}";
+  version = "20180209";
+
+  src = fetchurl {
+    url = "https://acpica.org/sites/acpica/files/acpica-unix-${version}.tar.gz";
+    sha256 = "1rpdfwa4vwnvyxdp9ygqjckmabc3s8kyg3jyq4n4f0rhr1zl4zy5";
+  };
+
+  NIX_CFLAGS_COMPILE = "-O3";
+
+  enableParallelBuilding = true;
+
+  buildFlags = stdenv.lib.concatStringsSep " " [
```

Comment:
Not hugely important, but is `concatStringsSep` really necessary? I'd expect it to end up being exported into the builder environment as a space separated string anyways.

Reply:
I think you're right, and the Nixpkgs manual needs some clarification.